content/guides/output-management.md

```diff
@@ -0,0 +1,73 @@
+---
+title: Output Management
```

Comment:
Hmm yeah, I see what you mean but I wanted to keep it a bit more general than that. Maybe __Output and Bundle Management__?